<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择汉堡</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择小食</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>90</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择饮料</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>134</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>巨无霸</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>134</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>麦辣鸡腿堡</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>134</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>培根蔬萃双层牛堡</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>134</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>中薯条</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>134</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>麦辣鸡翅</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_6">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>134</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>麦乐鸡块</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_7">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>190</y>
     <width>134</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>可乐</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_8">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>250</y>
     <width>134</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>雪碧</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
